common/utils/budget-sums.test.js

```diff
@@ -3679,6 +3687,841 @@ describe('budget summing methods', () => {
     });
   });
 
+  describe('sumShareCostsForCombinedTotals', () => {
+    test('with default values', () => {
+      const expected = {};
+      const actual = sumShareCostsForCombinedTotals();
+      expect(actual).toEqual(expected);
+    });
+
+    test('adds values to the combined totals for the year and overall total', () => {
+      const year = '2024';
+      const medicaidShare = 10;
+      const fedShare = 9;
+      const stateShare = 1;
+      const budget = {
+        combined: {
+          2024: {
+            medicaid: 100,
+            federal: 90,
+            state: 10
+          },
+          total: {
+            medicaid: 1000,
+            federal: 900,
+            state: 100
+          }
+        }
+      };
+      const expected = {
+        combined: {
+          2024: {
+            medicaid: 110,
+            federal: 99,
+            state: 11
+          },
+          total: {
+            medicaid: 1010,
+            federal: 909,
+            state: 101
```

Comment:
is total being stored too? or is that calculated somewhere else?

Reply:
Ah, `total.total` (and the `year.total`s) are being added via `sumTotalCostsByCategory`. Do you think there's a better name for the function to help clarify?